sources/dicom/large_image_source_dicom/__init__.py

```diff
@@ -110,11 +112,14 @@ def __init__(self, path, **kwargs):
         """
         super().__init__(path, **kwargs)
 
+        self.logger = config.getConfig('logger')
+
         # We want to make a list of paths of files in this item, if multiple,
         # or adjacent items in the folder if the item is a single file.  We
         # filter files with names that have a preferred extension.
+        # If the path is a dict, that likely means it is a DICOMweb asset.
         path = self._getLargeImagePath()
-        if not isinstance(path, list):
+        if not isinstance(path, (dict, list)):
```

Comment:
In the case that we have what looks like a string that isn't a path, should we see if the string is a json or yaml encoded dict (not on this PR)?  I think this might be needed if we are using a dicom web source as an entry in a multi-source source.

Reply:
This is tracked in #1310.